java-client/example-utilities/src/main/java/io/deephaven/client/examples/AuthenticationOptions.java

```diff
@@ -0,0 +1,37 @@
+package io.deephaven.client.examples;
+
+import picocli.CommandLine.Option;
+
+import java.util.function.Consumer;
+
+public class AuthenticationOptions {
+    @Option(names = {"--mtls"}, description = "Use the connect mTLS")
+    Boolean mtls;
+
+    @Option(names = {"--psk"}, description = "The pre-shared key")
+    String psk;
+
+    @Option(names = {"--explicit"}, description = "The explicit authentication type and value")
+    String explicit;
+
+    public String toAuthenticationTypeAndValue() {
+        if (mtls != null && mtls) {
+            // todo
```

Comment:
Is this a lost comment? Or should we add an issue tag?

Reply:
Meant to remove. We may eventually want proper string constants somewhere publicly accessible for authentication methods we support out of the box.